firebase-perf/src/main/java/com/google/firebase/perf/gauges/CpuGaugeCollector.java

```diff
@@ -46,10 +48,10 @@
 /** @hide */
 public class CpuGaugeCollector {
```

Comment:
You will also need to annotate this and the Memory class with `@Singleton` and make any other necessary change (if required) in the specific component/module.

Reply:
Done.